plugins/CoreHome/vue/src/QuickAccess/QuickAccess.vue

```diff
@@ -11,22 +11,23 @@
     class="quickAccessInside"
     v-focus-anywhere-but-here="{ blur: onBlur }"
   >
-    <span
-      class="icon-search"
-      @mouseenter="searchActive = true"
-    />
-    <input
-      class="s"
-      @keydown="onKeypress($event)"
-      @focus="searchActive = true"
-      v-model="searchTerm"
-      type="text"
-      tabindex="2"
-      v-focus-if="{ focused: searchActive }"
-      :title="quickAccessTitle"
-      :placeholder="translate('General_Search')"
-      ref="input"
-    />
+    <div v-tooltips="{ content: quickAccessTitle }">
```

Comment:
This actually doesn't work for me. It now simply doesn't show a tooltip at all.
Looking at how the directive works it tries to add a tooltip to all elements with title attribute. So I guess it should work correctly if you revert the changes here and just add the `v-tooltips` directive to the top element - without any parameters. It should then convert the title attribute on the input correctly.

Reply:
@sgiehl It’s updated. Sorry again.